x-pack/elastic-agent/pkg/composable/providers/kubernetes/config.go

```diff
@@ -25,6 +27,16 @@ type Config struct {
 
 	// Needed when resource is a Pod or Node
 	Node string `config:"node"`
+
+	AddResourceMetadata *metadata.AddResourceMetadataConfig `config:"add_resource_metadata"`
+	IncludeLabels       []string                            `config:"include_labels"`
+	ExcludeLabels       []string                            `config:"exclude_labels"`
```

Comment:
Where is `IncludeLabels` and `ExcludeLabels` used? I was not able to find them in the diff.

Reply:
These are passed and used deeper in meta Generators, which is already existent codebase.